docs/source/1.0/spec/core/model.rst

```diff
@@ -149,15 +149,32 @@ together to form a valid semantic model.
 Merging model files
 ===================
 
-Implementations MUST take the following steps to merge models together to load
-the semantic model:
+Implementations MUST take the following steps to merge model files together to
+form the semantic model:
+
+#. Merge the metadata objects of all model files using the steps defined in
+   :ref:`merging-metadata`.
+#. Shapes defined in a single model file are added to the semantic model as-is.
+#. Shapes with the same shape ID defined in multiple model files are merged
+   using the following rules:
+
+   #. All conflicting shapes MUST have the same shape type.
+   #. Conflicting :ref:`aggregate shapes <aggregate-types>` MUST contain the
+      same members that target the same shapes.
+   #. Conflicting :ref:`service shapes <service-types>` MUST contain the same
+      properties and target the same shapes.
+   #. The traits of conflicting shapes and the traits of their members are
```

Comment:
This doesn't seem right, isn't it an equality check not a merge? Except if they're using apply statements?

Reply:
The traits are merged, but the shape itself isn’t. I’ll try to make this more clear.